packages/factory_reset_tools/lib/pages/creation_progress_page.dart

```diff
@@ -24,6 +25,11 @@ class _CreationProgressPageState extends ConsumerState<CreationProgressPage> {
   );
 
   void onStatusChanged(ResetMediaCreationProgress progress) {
+    if (progress.status == ResetMediaCreationStatus.finished) {
+      Future.delayed(const Duration(seconds: 2), () {
```

Comment:
Is that delay intentional or left over from debugging?

Reply:
Intentional, otherwise you never see the last state 😅